packages/forms/resources/views/components/key-value.blade.php

```diff
@@ -25,32 +25,32 @@
                 'dark:divide-gray-700' => config('forms.dark_mode'),
             ])>
                 <thead>
-                    <tr @class([
```

Comment:
Could you fix these indentation changes?

Reply:
Yes, sure :)